pages/madewithcode.jsx

```diff
@@ -0,0 +1,82 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+var OutboundLink = require('react-ga').OutboundLink;
+
+var config = require('../lib/config');
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Made with Code</h1>
+        <section className="intro">
+          <Illustration
+          width={200} height={355}
+          src1x="/img/pages/madewithcode/logo_mwc.png"
+          alt="Woman training a young man on a computer"
+          className="content-first">
+            <h2>Welcome! Mozilla is happy to partner with Google's Made with Code initiative to inspire girls to get creative with code. Below are three fun activities to help you create your own webpages by writing and remixing HTML.</h2>
```

Comment:
@Pomax did you mean Made with Code should be wrapped in double quotes? ie. `“Made with Code”`